src/client/Microsoft.Identity.Client/Utils/CollectionHelpers.cs

```diff
@@ -25,6 +24,14 @@ public static IReadOnlyList<T> GetEmptyReadOnlyList<T>()
 #endif
         }
 
+#if HAVE_METHOD_IMPL_ATTRIBUTE
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+#endif
+        public static List<T> GetEmptyList<T>()
```

Comment:
Is this worth having? There's no interesting logic around it.

Reply:
Yea, I was thinking if there's more efficient way of creating an empty list, but since it's a concrete type, seems like no. Was leaning towards keeping this as a form of abstraction, but might remove it.